frontend/src/components/common/form/ContactInput.tsx

```diff
@@ -99,13 +99,38 @@ export const ContactInput: React.FC<ContactInputProps> = ({
 
 const StyledDiv = styled.div`
   border-radius: 0.3rem;
-  padding: 12px 6px 6px 12px;
+  padding: 6px;
   min-height: 2.5em;
   background-image: none;
-  background-color: ${props => props.theme.css.filterBackgroundColor};
+  color: #495057;
   border: #606060 solid 0.1rem;
 
   &.is-invalid {
     border: ${props => props.theme.css.dangerColor} solid 0.1rem;
   }
 `;
+
+export const StyledRemoveLinkButton = styled(LinkButton)`
+  &&.btn {
+    float: right;
+    color: #aaaaaa;
```

Comment:
same comment re: hardcoded colors vs trying one from CSS theme

Reply:
removed and used from themes